src/components/InstructorRevenueCard/components/LineChart/index.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import {map} from 'lodash'
+import hexToRgba from 'hex-rgba'
+import {Line} from 'react-chartjs-2'
+import numberFormattingByType from 'utils/numberFormattingByType'
+
+const sharedDataWithColor = (color) => ({
+  borderColor: color,
+  backgroundColor: hexToRgba(color, 20),
```

Comment:
Auto-calculate background fill as a light shade of main color:
![image](https://cloud.githubusercontent.com/assets/5497885/24528121/6d7d9196-1562-11e7-97ff-0e2eaec8ad41.png)
![image](https://cloud.githubusercontent.com/assets/5497885/24528124/715284de-1562-11e7-99e9-f9423a923bcc.png)